### PR TITLE
add labels to dependabot PRs

### PR DESCRIPTION
**Pull request by artis3n**

### CODE REVIEW

**Review by mamercad — CHANGES_REQUESTED** [review]
We need the [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-changelog-fragments) please :sammy-heart:

**Review by mamercad — APPROVED** [review]
🚀

File: changelogs/fragments/18-devcontainer.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+minor_changes:
```
[the remark this answers is not in the document]
Comment (artis3n) [reply changelogs/fragments/18-devcontainer.yaml:1]: Ah, good call out, thank you

**Review by mamercad — APPROVED** [review]
🚢

**Pull request merged**